### ISSUE TITLE
CommentedOutCode line comments factored into percentage

### ISSUE THREAD

**Issue by bkdotcom** [squizlabs/PHP_CodeSniffer#2844]
```php
/*
   // this is 100% valid commented code
   $foo = 1 / 0;   // divide by zero
*/
```
The inner comments should be discarded before calculating the percentage

Currently: 
"This comment is 30% valid code"

**Comment by gsherwood** [comment]
Strings that look like comments inside a comment block are already handled in a very specific way. Simply removing them would significantly increase the % values of code, producing a much greater number of false positives. Things like this:
```
/*
    There are a number of things to consider:
     #1. This one
     #2. This one
     #3. And this one

    Also note that this code: ++$foo instead of $foo++
    Fixes issue #2345 Some times of the bug report goes here.
*/
```
A change like this would need to be optional, so I think this would require a sniff property you can set if you'd prefer to have string that look like comments to not be included in calculations.
